lang/en/typeshed/stdlib/builtins.pyi

```diff
@@ -813,11 +1078,29 @@ class range(Sequence[int]):
     stop: int
     step: int
     @overload
-    def __init__(self, stop: SupportsIndex) -> None: ...
+    def __init__(self, stop: SupportsIndex) -> None:
```

Comment:
class range needs a top leve docstring, else we don't show it in the API documentation.